frontend/views/CreateGroup.vue

```diff
@@ -78,21 +78,24 @@
   .fade-leave-to /* .fade-leave-active below version 2.1.8 */ {
     opacity: 0;
   }
-</style>
-<style scoped>
+
```

Comment:
Any reason why this code is going to global css now?

Reply:
I will add the scoped back in, but also have added a bunch of comments about doing this properly with `transitions.js` (which currently has a poor implementation that isn't really being used), that's something that I hope you can improve.

Reply:
Created #529 to address this.